### PR TITLE
Feature/display nfts

### PR DESCRIPTION
**Pull request by nazariyv**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -283,31 +284,40 @@
...
           audiohandler
           id="jukebox"
-          scale="1.2 1.2 1.2"
-          gltf-model="#jukebox-model"
-          animation__mouseenter="property: scale; to: 1.5 1.5 1.5; dur: 300; startEvents: mouseenter"
-          animation__mouseleave="property: scale; to: 1.2 1.2 1.2; dur: 300; startEvents: mouseleave"
+          gltf-model="models/jukebox/scene.gltf"
+          scale="1.767 1.480 1.0"
+          position="-5.515 0 -8.713"
         >
```
Comment (microchipgnu) [inline index.html:305]: This shouldn't be deleted.

Comment (nazariyv) [reply index.html:305]: pardon didn't see this

Comment (microchipgnu) [reply index.html:305]: no worries

File: index.html [hunk 2]
```diff
@@ -299,9 +299,11 @@
...
           audiohandler
           id="jukebox"
-          gltf-model="models/jukebox/scene.gltf"
-          scale="1.767 1.480 1.0"
+          gltf-model="#jukebox-model"
           position="-5.515 0 -8.713"
+          scale="1.767 1.480 1.0"
+          animation__mouseenter="property: scale; to: 1.9 1.591 1.5; dur: 300; startEvents: mouseenter"
+          animation__mouseleave="property: scale; to: 1.767 1.480 1.075; dur: 300; startEvents: mouseleave"
         >
```
Comment (microchipgnu) [inline index.html]: LGTM!

**Comment by microchipgnu** [comment]
LGTM!

**Pull request merged**